rnd/autogpt_server/config.default.json

```diff
@@ -1,4 +1,5 @@
 {
   "num_graph_workers": 10,
-  "num_node_workers": 5
+  "num_node_workers": 5,
+  "num_user_credits_refill": 1500
```

Comment:
@ntindle 
Since we already have a default value in settings.py, is this actually needed?

Reply:
this is not technically, i was expecting us to config our servers this way rather than 1 trillion environment variables so we were going to have config.dev.json, config.staging.json etc and this was going to be the primary for those

Reply:
I see, should we consider merging it? Looping-in @aarushik93 here too.